tools/codegen/README.md

```diff
@@ -0,0 +1,53 @@
+# Weave SDK Code Generator
```

Comment:
I think it would be helpful to clearly define the steps if someone wants to add a new endpoint. A step-by-step guide would be great